### PR TITLE
set up icon with sidebar, let's add text in svelte

### PR DESCRIPTION
**Pull request by gladishd**

### CODE REVIEW

**Comment by gladishd** [comment]
approving new changes let us know if you want help running the extension !

File: package.json [hunk 1]
```diff
@@ -46,5 +67,10 @@
 		"mocha": "^8.2.1",
 		"typescript": "^4.1.3",
 		"vscode-test": "^1.5.0"
+	},
+	"dependencies": {
+		"@pyoner/svelte-ts-preprocess": "^1.3.0",
+		"rollup-plugin-svelte": "^6.0.0",
```
Comment (aitikgupta) [inline package.json:73]: Are we using rollup anywhere?

File: src/HomePanel.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ export class HomePanel {
     // Otherwise, create a new panel.
     const panel = vscode.window.createWebviewPanel(
       HomePanel.viewType,
-      "VSinder",
+      "Hello World",
```
Comment (aitikgupta) [inline src/HomePanel.ts:31]: Can we get this from the Webview layer instead of Extension layer?

File: src/extension.ts [hunk 3]
```diff
@@ -2,10 +2,18 @@
...
 import { HomePanel } from './HomePanel';
+import { SidebarProvider } from './SidebarProvider';
 
 // this method is called when your extension is activated
 // your extension is activated the very first time the command is executed
 export function activate(context: vscode.ExtensionContext) {
+	const sidebarProvider = new SidebarProvider(context.extensionUri);
+	context.subscriptions.push(
+		vscode.window.registerWebviewViewProvider(
+			"proj-sidebar",
```
Comment (aitikgupta) [inline src/extension.ts:13]: Is this the ID of the webview that we'll use on later?
Or is it just some random string

**Review by aitikgupta — COMMENTED** [review]
Leaving a few comments, since this is merged already, the changes can be taken up by another PR?

**Pull request merged**